sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -12,14 +12,13 @@ public enum ClientSideEncryptionVersion
     {
         V1_0 = 1,
     }
-    public partial class StorageCrc64NonCryptographicHashAlgorithm
+    public partial class StorageCrc64NonCryptographicHashAlgorithm : System.IO.Hashing.NonCryptographicHashAlgorithm
     {
-        internal StorageCrc64NonCryptographicHashAlgorithm() { }
-        public void Append(System.ReadOnlySpan<byte> source) { }
+        internal StorageCrc64NonCryptographicHashAlgorithm() : base (default(int)) { }
```

Comment:
Most of the other crypto algorithms, like https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.md5?view=net-6.0, have a public .ctor.  Any specific reason we're making this internal?

Reply:
Err, no, they have `protected` .ctors.  We can think about doing that as well in the future if we find a scenario.